app/controllers/root_controller.rb

```diff
@@ -109,12 +110,58 @@ def publication
       end
 
       @interaction_details = prepare_interaction_details(@publication, authority_slug, snac)
+    elsif @publication.format == 'local_transaction'
+      @location = fetch_location @postcode
+
+      if @location.nil?
+        @postcode_error = "fullPostcodeNoMapitMatch"
```

Comment:
I'm not sure that `"fullPostcodeNoMapitMatch"` is the right error here: `fetch_location` returns `nil` when no postcode is provided and doesn't talk to mapit at all.